src/actions/actions.js

```diff
@@ -418,6 +419,111 @@ export function markCellNotRendered() {
   }
 }
 
+function loginSuccess(userData) {
+  return (dispatch) => {
+    dispatch({
+      type: 'LOGIN_SUCCESS',
+      userData,
+    })
+    dispatch(updateAppMessages({ message: 'You are logged in' }))
+  }
+}
+
+function loginFailure() {
+  return (dispatch) => {
+    dispatch(updateAppMessages({ message: 'Login Failed' }))
+  }
+}
+
+export function login() {
+  if (window.location.hostname !== 'iodide.io') {
```

Comment:
Comment this out for testing the feature locally.

Reply:
Can you make this conditional on IODIDE_BUILD_MODE equaling heroku instead? This would allow us to use/test this functionality in dev/staging/etc environments (e.g. your own iodide-sv heroku environment) without code modifications. Can see some examples of this in the store code:

https://github.com/iodide-project/iodide/blob/14184cbbcf645ec459432e5634d683fbc2e74a58/src/store.js#L15